### PR TITLE
Added release v1.9.1 notes [6331]

### PR DESCRIPTION
**Pull request by LuisGP**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/44/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/45/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/46/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/50/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/51/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/52/)

File: docs/notes.rst [hunk 1]
```diff
@@ -1,8 +1,38 @@
-Version 1.9.0
+Version 1.9.1
 =============
 
 This release includes the following features:
 
+* Fast-RTPS-Gen is now an independent project.
+* Header **eClock.h** marked is now deprecated.
```
Comment (raquelalvarezbanos) [inline docs/notes.rst]:
```suggestion
* Header **eClock.h** is now marked as deprecated.
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/54/)

File: docs/notes.rst [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+
+It also adds the following bug fixes and improvements:
+
+* Bump to FastCDR v1.0.11.
+* Installation from sources documentation fixed.
+* Fixed assertion on WriterProxy.
+* Fixed potential fall through while parsing Parameters.
+* Removed deprecated guards causing compilation errors in some 32 bits platforms.
+* Improve windows performance by avoiding usage of *_Cnd_timedwait* method.
+* Multicast is sent through *localhost* too.
```
Comment (raquelalvarezbanos) [inline docs/notes.rst]:
```suggestion
* Fixed reported communication issues by sending multicast through *localhost* too.
```

File: docs/notes.rst [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+* Fixed potential race conditions and deadlocks.
+* Eliminating use of *acceptMsgDirectTo*.
+* Discovery Server framework reconnect/recreate strategy.
+* Removed unused folders.
+* Restored subscriber API.
+* SequenceNumber_t improvements.
```
Comment (raquelalvarezbanos) [inline docs/notes.rst]: I miss some improvements/fixes:

* dll export of addToCDRMessage()
* Improved XML profile example (not sure this one is important enough to be mentioned though)
* Improved blackbox tests that were failing frequently
* Fixes to latency and throughtput tests
* Added STRICT_REALTIME cmake option

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/55/)

File: docs/notes.rst (hunk 3 above)
Comment (LuisGP) [reply docs/notes.rst]: > I miss some improvements/fixes:
> 
> * dll export of addToCDRMessage()

This one appears as `Restored subscriber API.`

> * Improved XML profile example (not sure this one is important enough to be mentioned though)

I don't think examples improvements are important enough to be mentioned.

> * Improved blackbox tests that were failing frequently
> * Fixes to latency and throughtput tests

We agreed to don't add tests' related changes.

> * Added STRICT_REALTIME cmake option

That change wasn't present at the time the document was updated.

> Is there a reason why they were not added?

I guess we should add the STRICT_REALTIME change, but, what about the others?

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/56/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/57/)

**Pull request merged**